### PR TITLE
fix: csr properly ported in cli flow

### PR DESCRIPTION
**Pull request by dnitsch**
#### 📲 What

Bring CSR up to par with other project types

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: packages/scaffolding-cli/src/__tests__/domain/workers/main_worker.test.ts [hunk 1]
```diff
@@ -163,19 +163,13 @@ describe("mainWorker class tests", () => {
...
-                staticConf.csr.gitRef,
-            )
             expect(Utils.prepBase).toHaveBeenCalled()
             expect(Utils.constructOutput).toHaveBeenCalled()
             expect(flow_ran).toHaveProperty("message")
             expect(flow_ran).toHaveProperty("ok")
             expect(flow_ran.ok).toBe(true)
             expect(flow_ran.message).toMatch(
-                `cd ${mockAnswerSsr.projectName}/src && npm install && npm run stuff`,
+                `cd ${mockAnswerSsr.projectName}/src && npm install && npm run build && npm run start`,
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/src/__tests__/domain/workers/main_worker.test.ts:172]: For local, just need `npm install && npm run start`

File: packages/scaffolding-cli/src/domain/config/static.config.json [hunk 2]
```diff
@@ -199,8 +179,8 @@
     ]
   },
   "javaSpring": {
-    "gitRepo": "https://github.com/spring-guides/gs-actuator-service.git",
-    "gitRef": "17bbbb65c1c641634a59568c2e3642ae59e2e7dd",
+    "gitRepo": "https://github.com/amido/stacks-java.git",
+    "gitRef": "initial-commit",
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/src/domain/config/static.config.json:183]: There isn't a master branch or init tag we could create?

Comment (dnitsch) [reply packages/scaffolding-cli/src/domain/config/static.config.json:183]: I think we should let the Java team control this going forward - @peet-j @ElvenSpellmaker

Comment (ElvenSpellmaker) [reply packages/scaffolding-cli/src/domain/config/static.config.json:183]: Once we merge the initial-commit branch into master (hopefully today) I'll see about tagging a 0.0.1 initial release, we may wait until it's more fleshed out though. 🙂

**Pull request merged**